packages/flutter/lib/src/material/menu_anchor.dart

```diff
@@ -321,6 +328,10 @@ class _MenuAnchorState extends State<MenuAnchor> {
       _internalMenuController = MenuController();
     }
     _menuController._attach(this);
+    _animateController = AnimationController(
+      duration: const Duration(milliseconds: 500),
```

Comment:
Can we make this duration a constant?

Reply:
Removed this one as it is unnecessary here.